backend/app/services/streaming.py

```
@@ -0,0 +1,342 @@
+        try:
+            data = {
+                "step": step,
+                "progress": progress,
+                "message": message,
+                "correlation_id": correlation_id,
+                **(additional_data or {})
+            }
+
+            await self.publish_event(tenant_id, project_id, "progress", data)
+
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)